e2e/README.md

```
@@ -0,0 +1,11 @@
+# Interchaintest
+
+## Current ICTests
+| CLI | Description   |  |   |   |
```

Markdownlint (0.35.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)